woocommerce/payment-gateway/class-sv-wc-payment-gateway-my-payment-methods.php

```diff
@@ -795,21 +779,6 @@ protected function get_table_body_row_data( $token ) {
 	}
 
 
-	/**
-	 * Get the payment method title for a given token.
-	 *
-	 * @since 4.0.0
-	 * @deprecated 5.1.0
-	 *
-	 * @param SV_WC_Payment_Gateway_Payment_Token $token token object
-	 * @return string
-	 */
-	protected function get_payment_method_title( $token ) {
-
-		return $this->get_payment_method_title_html( $token );
-	}
-
-
```

Comment:
Out for curiosity, why did you remove this one instead of deprecating it? I noticed you deprecated at least one other protected method...

Reply:
That one was deprecated in February 2018, but we didn't add a notice or a TODO, only the deprecated tag. Since it's been almost two years, I think is ok to remove it.